libdnf/package/modulepackage.cpp

```diff
@@ -0,0 +1,25 @@
+#include "modulepackage.hpp"
```

Comment:
Can you split this into another commit? It has nothing to do with adding tests (AFAIK).

Reply:
Those are methods that are used for testing.